pytorch_transformers/tokenization_xlm.py

```diff
@@ -187,19 +315,34 @@ def bpe(self, token):
         self.cache[token] = word
         return word
 
-    def _tokenize(self, text):
+    def _tokenize(self, text, lang='en'):
```

Comment:
We should fix the docstring of `XLMTokenizer` which is now totally wrong:
- remove the reference to SpaCy
- explain that people need to input a `lang` argument to the `tokenize` and `encode` function.

The `encode` method currently doesn't accept `kwargs`. I will add that and make them forwarded to `tokenize` so the user can do `tokenizer.encode(text, lang='jp')` as well.